exporter.go

```diff
@@ -562,21 +564,24 @@ func (l *StatsDTCPListener) Listen(e chan<- Events) {
 func (l *StatsDTCPListener) handleConn(c *net.TCPConn, e chan<- Events) {
 	defer c.Close()
 
+	tcpConnections.Inc()
+
 	r := bufio.NewReader(c)
 	for {
 		line, isPrefix, err := r.ReadLine()
 		if err != nil {
 			if err != io.EOF {
-				networkStats.WithLabelValues("tcp_error").Inc()
+				tcpErrors.Inc()
 				log.Debugf("Read %s failed: %v", c.RemoteAddr(), err)
 			}
 			break
 		}
 		if isPrefix {
-			networkStats.WithLabelValues("tcp_line_too_long").Inc()
+			tcpLineTooLong.Inc()
```

Comment:
Uhm, we should just fix the reader though, no need to abort on a long line. https://golang.org/pkg/bufio/#Reader.ReadLine